project/src/analysis/sampler.py

```diff
@@ -0,0 +1,27 @@
+"""Use the pymbar package to perform decorrelated equilibration sampling"""
+
+import numpy as np
+from pymbar import timeseries
+
+
+def decorr_sampling(job, data_type="potential_energy"):
```

Comment:
we might break this into two functions to make testing easier. e.g., one function handles the job and the other just wraps the equilibration/decorrelation. ([Here's an example of what I mean from the rdf PR](https://github.com/mosdef-hub/reproducibility_study/pull/35/commits/39860a633ff97a8e177980fb82b9abec14cd554c) thanks to a suggestion from @justinGilmer)

Reply:
Ok, this is done. Let me know if this is what you had in mind.

Reply:
LGTM c: